### ISSUE TITLE
Study Tab

### ISSUE THREAD

**Issue by laura-chicas** [chicas-lancaster/websource#40]
We would like to add a new tab at the top of the CHICAS website entitled 'Study'.

**Comment by barryrowlingson** [comment]
Whereabouts in the menu should it go?

Current list is:

    Home
    About Us
    Projects
    People
    Research Themes
    News
    Events

**Comment by laura-chicas** [comment]
Just a few thoughts for now... Based on Engineering and LMS sites, I would probably change the order as follows to fall more in line with the University's webpages:
Home
Study
Research
Projects
About Us
People
News
Events

I don't suppose there's a way in which we can remove the projects tab and just have the projects connected to the people/research? The Events tab doesn't currently have anything in - there are things under News which could in my opinion be classed as events, or we could just do away with the Events tab and put everything under News - what do you think? It's a shame we can't have something more visible on the main page showing events really - if there's enough events to make the Events tab work, then making it more prominent would work more effectively I think, i.e. to have the events showing on the main page instead of having to click on a tab to see what they are. Though this would also depend on what sort of events we're holding, who the attendees are and whether these need to be visible to the public.

**Comment by barryrowlingson** [comment]
We can do almost anything with the main menu... We can also redo the boxes at the bottom - the map is cute but I'm not sure it gets much use, and the "Work with" section doesn't do an awful lot for the space it has.

**Comment by barryrowlingson** [comment]
If we have upcoming events they would show up in the sidebar with the News. The difference between News and Events should be that Events should be something happening on a calendar day or days and News should be announcements of interesting things that have already happened...

**Comment by laura-chicas** [comment]
Yes I think perhaps one idea would be to remove the map and move the 'work with CHICAS' bit to a tab at the top which could be called 'CHICAS partners', if the study info is moved to a 'study' tab then this frees up the space at the bottom. I am working with the PGR Student Rep to arrange a Twitter feed which would be great if it could appear to the right where 'latest news' currently sits. However, if we have upcoming news on the feed, perhaps this could replace the News/events section and we could instead have these as tabs which you can then click on if you're looking for a particular news item/event - I'm really sure how to do this though, what are your thoughts?

**Comment by barryrowlingson** [comment]
I'm amenable to any changes to the home page, the only tricking things would be big changes to the structure or layout - for example if you wanted it to look more like the university pages! Changing the content within the structure is usually easy enough. I'm not sure about embedding a twitter feed in a page, I did it a few years ago for another site so it shouldn't be too hard.

In an email from Emanuele he said to leave home page discussion to the next CHICAS meeting, so I'm happy to park this here for now.

**Comment by laura-chicas** [comment]
No problem, that's fine though do you think we could see if we could try and improve the logos for now, if that's easy enough to change?

**Comment by laura-chicas** [comment]
Hi Barry, I've just been looking back on some old emails and wondered if we could pick up on the research themes on the CHICAS website. I was wondering whether we could work out which research theme each of the projects would come under and then arrange them so that when you click on the research themes tab, and then click on the title for each of the research themes, it leads to the projects. I also wondered whether it would be a good idea to have a list of staff under each of the research themes - and then their names linked to their profile pages. I see that the staff have their projects listed on their pages also but I think it would be useful for folks who browse the projects first to know who is involved with them and so can go directly to the member of staff. I'm thinking potential PhD applicants looking for supervisors for example. It would make sense for a PhD applicant to search first on the area of interest and then be able to navigate to those who are involved in that area of interest.

**Comment by barryrowlingson** [comment]
Had a think and written this proposal spec:

Research Theme Website Proposal
===============================

Main menu "Research Themes": change to link to:

https://chicas.lancaster-university.uk/themes/

That page lists research themes with an image and a short (one-liner) description, with links to individual theme pages:

https://chicas.lancaster-university.uk/themes/[research theme]/

...is the page for the theme - has full description of the theme, images, and sidebar with:

 *  people profiles - (don't show former people)
 *  list of projects - (don't include expired projects)
 *  external links.

Questions:
----------

Do we want people profile pages to show research theme memberships?
This could go under the profile picture and personal details before the
person's "Current Projects" list.

Do we want project web pages to show research theme memberships?
Could go as a sub-heading under the main group heading, linking back
to the research theme page.

Open Ideas
----------

More distinctive branding of research themes - logos, icons, colours etc.
Maybe define all research themes with a three-letter abbreviation and
badge the Chicas logo with it. Probably taken this too far now...


Underlying Mechanics
--------------------

These are notes for me on how I'd implement this in the content management
system. This is mostly meaningless to anyone else...

### Need

theme index page and template:
 * gets content from theme page metadata

theme pages and template:
  * has title, description
  * displays content block, images, sidebar

assignment of projects to themes:

  * either include projects in theme metadata or include theme in project metadata. not sure which is easier...

assignment of people to themes:

  * people are assigned to projects via people: metadata in the project metadata,
  so probably easiest to have a people: section in the theme metadata. Can then
  re-use code.
